docs/detections/alerts-view-details.asciidoc

```diff
@@ -123,33 +181,114 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+* **Alerts related by source event**: Shows the number of alerts that were created by the same source event.  
+* **Cases related to the alert**: Shows the number of cases to which the alert has been added.
+* **Alerts related by session ID**: Shows the number of alerts that generated during the same session. 
+* **Alerts related by process ancestry**: Shows the number of alerts that are related by process events on the same linear branch. 
++
+NOTE: To access data about alerts related by process ancestry, you must have a https://www.elastic.co/pricing[Platinum or higher subscription].
+
+[discrete]
+[[expanded-correlations-view]]
+==== Expanded correlations view
 
-Available threat indicator match data appears in the **Threat Match Detected** subsection.
+From the right panel, click **Correlations** to open the expanded Correlations view within the left panel. 
 
 [role="screenshot"]
-image::images/matched-indicator-sub-sec.png[Threat Match Detected subsection with matched file hash, 600]
+image::images/expanded-correlations-view.png[Expanded view of correlation data, 65%]
+
+In the expanded view, corelation data is organized into several tables:
+
+* **Suppressed alerts**: Shows how many duplicate alerts were suppressed. This information only appears if alert suppression is enabled for the rule.
++
+preview::[]
+
+* **Related cases**: Shows cases to which the alert has been added. Click a case's name to open its details.
+* **Alerts related by source event**: Shows alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
+* **Alerts related by session**: Shows alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+* **Alerts related by ancestry**: Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
+
+[discrete]
+[[prevalence-overview]]
+=== Prevalence
+
+The Prevalence overview shows whether data from the alert was frequently observed on other host events from the last 30 days. Prevalence calculations use values from the alert’s highlighted fields and results are displayed as percentages. Highlighted field values are considered prevalent, or _common_, if they're observed on over 10% of hosts in your environment. Highlighted field values are considered not prevalent, or _uncommon_, if they're observed on less than 10% of hosts in your environment. 
```

Comment:
```suggestion
The Prevalence overview shows whether data from the alert was frequently observed on other host events from the last 30 days. Prevalence calculations use values from the alert’s highlighted fields. Highlighted field observed on less than 10% of hosts in your environment are considered not prevalent, or _uncommon_, and are therefore displayed in the Prevalence overview. Highlighted field values observed on over 10% of hosts in your environment are considered prevalent, or _common_, and are not listed in the Prevalence overview. 
```

Reply:
Making some minor editorial tweaks to the edited sentences. 

```suggestion
The Prevalence overview shows whether data from the alert was frequently observed on other host events from the last 30 days. Prevalence calculations use values from the alert’s highlighted fields. Highlighted field values that are observed on less than 10% of hosts in your environment are considered uncommon (not prevalent) and are mentioned in the Prevalence overview. Highlighted field values that are observed on more than 10% of hosts in your environment are considered common (prevalent) and are not mentioned in the Prevalence overview. 
```